tox.ini

```diff
@@ -6,6 +6,7 @@ passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST
 extras =
   tests
   dali
+  haydn_op20
```

Comment:
did you mean `music21`?

Reply:
Hi @magdalenafuentes 
Haydn is the alias not the library :). I think it is correct

extras_require={
            "tests": [
                "pytest>=4.4.0",
                "pytest-cov>=2.6.1",
                "pytest-pep8>=1.0.0",
                "pytest-mock>=1.10.1",
                "pytest-localserver>=0.5.0",
                "testcontainers>=2.3",
                "future==0.17.1",
                "coveralls>=1.7.0",
            ],
            "docs": [
                "numpydoc",
                "recommonmark",
                "sphinx>=3.4.0",
                "sphinxcontrib-napoleon",
                "sphinx_rtd_theme",
            ],
            "dali": ["dali-dataset==1.1"],
            "haydn_op20": ["music21"],
        },